packages/gatsby/src/utils/gatsby-dependents.js

```diff
@@ -0,0 +1,12 @@
+const util = require(`util`)
+const readPackageTree = require(`read-package-tree`)
```

Comment:
https://github.com/npm/read-package-tree seems to have a promise api itself.

also don't forget to add it to package.json